src/lib.rs

```diff
@@ -136,18 +139,22 @@ impl<E: std::error::Error> miniquad::EventHandlerFree for EventHandlerWrapper<E>
     }
 }
 
-pub fn start<F, E>(conf: conf::Conf, f: F) -> GameResult
+pub fn start<F, E>(quad_conf: miniquad::conf::Conf, conf: conf::Conf, f: F) -> GameResult
```

Comment:
I do not recommend using miniquad's Conf in public API. 

Cache, for example, is gwg-only thing and should be removed from miniquad's Conf.

Reply:
(cache in miniquad is a legacy from times when miniquad was not able to load files by url)

Reply:
Ah, I didn't know that. Thx for explaining.
Then I'll put cache back into the good-web-game `Conf`. It fits better anyway.
I'll probably just add all the miniquad fields to it as well and then simply create a miniquad Conf to hand over when starting the application.